recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+#include "bcm2712-rpi-5-b.dts"
+
+/ {
+	chosen {
+		#address-cells = <0x2>;
+		#size-cells = <0x2>;
+		xen,xen-bootargs = "console=dtuart dtuart=/soc/serial@7d001000 dom0_mem=128M dom0_max_vcpus=1 xsm=flask flask=permissive";
+		xen,dom0-bootargs = "console=hvc0 earlycon=xen earlyprintk=xen clk_ignore_unused root=/dev/ram0";
+		xenpolicy {
+			compatible = "xen,xsm-policy", "xen,multiboot-module";
+		};
+		dom0 {
+			compatible = "xen,linux-zimage", "xen,multiboot-module", "multiboot,module";
+		};
+		domU0 {
```

Comment:
We agreed you rename domU0 to domD here also.

Reply:
Done